groups/chd-infrastructure/locals.tf

```diff
@@ -0,0 +1,46 @@
+# ------------------------------------------------------------------------
+# Locals
+# ------------------------------------------------------------------------
+locals {
+  admin_cidrs = values(data.vault_generic_secret.internal_cidrs.data)
+
+  internal_fqdn = format("%s.%s.aws.internal", split("-", var.aws_account)[1], split("-", var.aws_account)[0])
+
+  default_tags = {
+    Terraform = "true"
+    Region    = var.aws_region
+    Account   = var.aws_account
+  }
+
+# ------------------------------------------------------------------------------
+# CHD BEP
+# ------------------------------------------------------------------------------
+#  chd_bep_data = data.vault_generic_secret.chd_bep_data.data_json
+  chd_ec2_data = data.vault_generic_secret.chd_ec2_data.data
+  s3_releases  = data.vault_generic_secret.s3_releases.data
+
+  kms_keys_data          = data.vault_generic_secret.kms_keys.data
+  security_kms_keys_data = data.vault_generic_secret.security_kms_keys.data
+  logs_kms_key_id        = local.kms_keys_data["logs"]
+  ssm_kms_key_id         = local.security_kms_keys_data["session-manager-kms-key-arn"]
+
+  security_s3_data            = data.vault_generic_secret.security_s3_buckets.data
+  session_manager_bucket_name = local.security_s3_data["session-manager-bucket-name"]
+
+  bep_cw_logs = { for log, map in var.bep_cw_logs : log => merge(map, { "log_group_name" = "${var.application}-bep-${log}" }) }
+  bep_log_groups = compact([for log, map in local.bep_cw_logs : lookup(map, "log_group_name", "")])
+
+  chd_bep_ansible_inputs = {
+    s3_bucket_releases         = local.s3_releases["release_bucket_name"]
+    s3_bucket_configs          = local.s3_releases["config_bucket_name"]
+    heritage_environment       = var.environment
+    version                    = var.bep_app_release_version
+    default_nfs_server_address = var.nfs_server
+    mounts_parent_dir          = var.nfs_mount_destination_parent_dir
+    mounts                     = var.nfs_mounts
```

Comment:
var.nfs_mounts isn't passed in anywhere i can see, so this will pass in the initial mount used as a test that is set as the default, not sure if this was intentional?

Reply:
This is intended as NFS mounts have yet to be defined. The NFS role is also currently disabled in the initial AMI builds to prevent false-positives from the watcher script.